### PR TITLE
Improve performance of renconciliation by adding an option to show or not letter items

### PR DESCRIPTION
**Pull request by thib44**

### CODE REVIEW

File: app/models/preference.rb [hunk 1]
```diff
@@ -45,7 +45,7 @@ class Preference < Ekylibre::Record::Base
                             decimal language integer record
                             spatial_reference_system string], predicates: true
   @@conversions = { float: :decimal, true_class: :boolean, false_class: :boolean, fixnum: :integer }
-  cattr_reader :reference
+  attr_reader :reference
```
Comment (Aquaj) [inline app/models/preference.rb]: `@@reference` is a class variable, why did you change this to be `attr_reader` instead of the previously `cattr_reader`?

File: Gemfile.lock [hunk 2]
```diff
@@ -24,7 +24,7 @@ GEM
       rails-html-sanitizer (~> 1.0, >= 1.0.3)
     active_list (6.9.3)
       arel (>= 5.0.0)
-      code_string (>= 0.0.0)
```
Comment (burisu) [inline Gemfile.lock]: Please `git checkout master` this file

File: app/views/backend/accounts/mark.html.haml [hunk 3]
```diff
@@ -39,28 +39,10 @@
...
-                - unless item.letter.blank? or item.closed?
-                  = link_to({action: :unmark, id: @account.id, letter: item.letter}, class: "unmark", method: :post, data: {confirm: :are_you_sure.tl}) do
-                    %i
-                    = :unmark.ta
-              %td.dat{conf}= item.printed_on.l
-              %td.code{conf}= link_to(item.entry_number, {action: :show, controller: :journal_entries, id: item.entry_id}, keep: true)
-              %td{conf}= item.name
-              %td.dec.debit{conf}=  number_to_accountancy(item.absolute_debit, currency)
-              %td.dec.credit{conf}= number_to_accountancy(item.absolute_credit, currency)
+            = render('backend/accounts/journal_entry_item', item: item, currency: currency)
```
Comment (burisu) [inline app/views/backend/accounts/mark.html.haml]: `render('journal_entry_item', item: item, currency: currency)` should be enough

**Comment by Aquaj** [comment]
Fixed elsewhere
